### DISCUSSION TITLE
New Flag to be introduce with --nodatavalidate ( or something like this ) to not validate the content of the Repository while retrieve from org

### DISCUSSION THREAD

**Discussion by vishalcopado** [forcedotcom/cli#2348] [Ideas]
**Is your feature request related to a problem? Please describe.**
Currently , If in my  CustomLabel i have any label1 which is not present to my environment 1 to which I have connected my repository , but it is present in another environment where I have previous selected connected my repository so lebel1 is not present in our org i am connected with but was present in another org where i was earlier connect. Now when I try to retrieve all the CustomLabels in my connected org , the retrieve command validate the content present in my repository and see that label1 is not present as an entity in org. I want a feature flag saying dont validate the content of the repository , only retrieve the selected metadata.

**What are you trying to do**
Expecting a new flag sfdx force:source:retrieve -m CustomLabels:CustomLabels --nodatavalidate , which will not validate data present in the repo and will only retrieve 

**Describe the solution you'd like**
When I run the command with this new flag , cli should not validate the data present in the repo rather it just retrieve things from the organization.

**Describe alternatives you've considered**
No Alternatives for now , have to manually remove the unwanted data from the repo

**Additional context**

[a comment by an automated account is not shown]

**Comment by vishalcopado** [comment]
any update on this work item ?

**Comment by WillieRuemmele** [comment]
I'm quite confused on what you're trying to achieve with this. If you want to overwrite your local CustomLabels file, you can delete the file, and retrieve the new one from the new org?  If you're doing a `force:source:pull` or `project retrieve start` you'll be able to force override conflicts and write what's retrieved.
